template_debian/09_cleanup.sh

```diff
@@ -26,13 +26,16 @@ rm -rf "${INSTALLDIR}/var/cache/apt/srcpkgcache.bin"
 rm -f "${INSTALLDIR}/etc/apt/sources.list.d/qubes-builder.list"
 rm -rf "${INSTALLDIR}/${TMPDIR}"
 rm -f "${INSTALLDIR}/var/lib/systemd/random-seed"
-rm -f "${INSTALLDIR}/var/lib/exim4/config.autogenerated"
-rm -f "${INSTALLDIR}/etc/mailname"
-sed -i /dc_other_hostnames=.*/d  "${INSTALLDIR}/etc/exim4/update-exim4.conf.conf"
-sed -i '/./{H;$!d};x;/dc_other_hostnames/d'  "${INSTALLDIR}/var/cache/debconf/config.dat"
-sed -i '/./{H;$!d};x;/exim4\/mailname/d'  "${INSTALLDIR}/var/cache/debconf/config.dat"
-sed -i '/./{H;$!d};x;/dc_other_hostnames/d'  "${INSTALLDIR}/var/cache/debconf/config.dat-old"
-sed -i '/./{H;$!d};x;/exim4\/mailname/d'  "${INSTALLDIR}/var/cache/debconf/config.dat-old"
+if [ "$DIST" == "jessie" ]; then
+    rm -f "${INSTALLDIR}/var/lib/exim4/config.autogenerated"
+    rm -f "${INSTALLDIR}/etc/mailname"
+    sed -i /dc_other_hostnames=.*/d  "${INSTALLDIR}/etc/exim4/update-exim4.conf.conf" || true
+    sed -i '/./{H;$!d};x;/dc_other_hostnames/d'  "${INSTALLDIR}/var/cache/debconf/config.dat" || true
+    sed -i '/./{H;$!d};x;/exim4\/mailname/d'  "${INSTALLDIR}/var/cache/debconf/config.dat" || true
+    sed -i '/./{H;$!d};x;/dc_other_hostnames/d'  "${INSTALLDIR}/var/cache/debconf/config.dat-old" || true
+    sed -i '/./{H;$!d};x;/exim4\/mailname/d'  "${INSTALLDIR}/var/cache/debconf/config.dat-old" || true
```

Comment:
Here? or in file?

The target files contain paragraphs separated by blank lines.
`sed -i '/./{H;$!d};x;/dc_other_hostnames/d'` changes the file in place. (switch -i)

`/./ `matches a non-empty line.
`H` adds that line to the hold buffer. `$!d `suppresses printing the line
When an empty line is found "`x`"  moves contents of hold buffer to pattern buffer.
If the pattern buffer contains dc_other_hostnames: "d" deletes the paragraph
Otherwise move to next line and repeat